--- packages/e2e/scenarios/api.test.ts
+++ packages/e2e/scenarios/api.test.ts
@@ -42,6 +42,9 @@

     const res = await request.post(`${API_URL}/gql`, {
+      headers: {
+        Authorization: 'Bearer dGhpcyBpcyBhbiBhbm9ueW1vdXMgdXNlcg==',
+      },
       data: { query },
     });
     const json = await res.json();
@@ -69,6 +72,9 @@

     const res = await request.post(`${API_URL}/gql`, {
+      headers: {
+        Authorization: 'Bearer dGhpcyBpcyBhbiBhbm9ueW1vdXMgdXNlcg==',
+      },
       data: { query },
     });
     const json = await res.json();
@@ -111,6 +117,9 @@

     const res = await request.post(`${API_URL}/gql`, {
+      headers: {
+        Authorization: 'Bearer dGhpcyBpcyBhbiBhbm9ueW1vdXMgdXNlcg==',
+      },
       data: { query },
     });
     const json = await res.json();
